configuration/configuration.go

```diff
@@ -57,6 +57,9 @@ type Configuration struct {
 			// contain the private portion for the file specified in
 			// Certificate.
 			Key string `yaml:"key,omitempty"`
+
+			// Specifies the CA cert for client authentication
+			ClientCAs []string `yaml:"ca,omitempty"`
```

Comment:
The yaml tag should be "clientcas". It needs to be clear that this is the CA for client certificates.